dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -580,12 +633,12 @@ def test_profile(self):
                 'bin_edges': np.array([2.5, 5.0, 7.5, 10.0, 12.5]),
             },
             quantiles={
-                0: 3.75,
-                1: 5.0,
-                2: 10.0
+                0: 2.5075,
+                1: 5.005 ,
```

Comment:
idk if we need the exra space, but :man_shrugging:

Reply:
previously there were comments after and I was lining them up, but makes sense to not need now